yb-voyager/src/migassessment/sizing.go

```diff
@@ -405,11 +476,11 @@ Returns:
 	float64: The total size of the source database in gigabytes.
 */
 func getSourceMetadata() ([]SourceDBMetadata, []SourceDBMetadata, float64) {
```

Comment:
handle the error returned by `SourceMetaDB.Close()` function

Reply:
cc @shaharuk-yb

Reply:
handled the error